faq.md

```diff
@@ -1,112 +1,125 @@
 # RChain Cooperative FAQ
 
-    Note: the FAQ will get an update to reflect decisions made in the Cooperative Members Meeting
-    held on October 24th.
-
 ### Sections
 
-[General](#general)  
-[Cooperative](#cooperative)  
-[Tokens](#tokens)  
-[Activists](#activists)  
-[Developers](#developers)  
+[General](#general)  \
+[Cooperative](#cooperative)  \
+[Tokens](#tokens)  \
+[Members](#members)  \
+[Developers](#developers)
 
 ## General
 
-**What is RChain?**  
+**What is RChain?**  \
 RChain is a fundamentally new blockchain platform rooted in a formal model of concurrent and decentralized computation. The RChain Cooperative is leveraging that model through correct-by-construction software development to produce a concurrent, compositional, and infinitely scalable blockchain.
 
-**Where is the whitepaper?**  
+**Where is the whitepaper?**  \
 [On Github (PDF)](http://docs.google.com/gview?url=https://github.com/rchain/reference/raw/master/docs/RChainWhitepaper.pdf).
 See also the [RChain Architecture](http://rchain-architecture.readthedocs.io).
 
-**Where is the roadmap?**  
-A snapshot of the platform [milestone definitions](https://docs.google.com/spreadsheets/d/1_cPts5BdoZVmbikDGUhkgkdQvkeXbP6aKNI31O0BV3E/edit?usp=sharing) is available for commentary. The team has not yet committed to specific schedule dates. One estimate is the first release, Mercury, will be available after the goal of $10M committed investment is achieved.
+**Where is the roadmap?**  \
+A snapshot of the platform [milestone definitions](https://docs.google.com/spreadsheets/d/1_cPts5BdoZVmbikDGUhkgkdQvkeXbP6aKNI31O0BV3E/edit?usp=sharing) is available for commentary. The team has not yet committed to specific schedule dates.
```

Comment:
That's fairly out-of-date and doesn't really break things down in the way they're actually being worked on. There is a deadline of one year from close of the private sale for the delivery of Mercury. Medha can point you to stuff on JIRA that describes what the plan is for Mercury and where we are in implementing it.  There's a graphic at the top of https://rchain.atlassian.net/wiki/spaces/CORE/overview that gives a 50,000-foot view.